example/trace/http/client/client.go

```diff
@@ -49,13 +48,13 @@ func initTracer() func() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	return flush
+	return tp, flush
 }
 
 func main() {
-	flush := initTracer()
+	tp, flush := initTracer()
 	defer flush()
-	tr := global.TraceProvider().Tracer("cloudtrace/example/client")
+	tr := tp.Tracer("cloudtrace/example/client")
 
 	client := http.DefaultClient
 	ctx := correlation.NewContext(context.Background(),
```

Comment:
Not related to this PR. but do attributes we add to the correlation context show up anywhere? My understanding is that this is not implemented in the SDK yet, so perhaps we should remove it from the examples for now.